packager/packager.cc

```diff
@@ -174,15 +174,15 @@ class PackageDB final {
         std::string_view path = ctx.file.data(ctx).path();
         int curPrefixPos = path.find_last_of('/');
         while (curPrefixPos != std::string::npos) {
-            const auto &it = packageInfoByPathPrefix.find(path.substr(0, curPrefixPos + 1));
+            const auto it = packageInfoByPathPrefix.find(path.substr(0, curPrefixPos + 1));
             if (it != packageInfoByPathPrefix.end()) {
                 return it->second.get();
             }
 
             curPrefixPos = path.find_last_of('/', curPrefixPos - 1);
         }
 
-        const auto &root = packageInfoByPathPrefix.find("");
+        const auto root = packageInfoByPathPrefix.find("");
```

Comment:
again just ooc: why are we dropping the `&`s here?

Reply:
`find` returns an iterator value, so it's not clear to me that it's good to declare these as references to intermediates.